cw/capture.py

```diff
@@ -135,6 +135,7 @@ def capture_init(ctx, num_traces, plot_traces):
 
     # Key and plaintext generator
     ctx.obj.ktp = cw.ktp.Basic()
+    ctx.obj.ktp.fixed_key = False # if true, then key never updates and setting key_len has no effect
```

Comment:
This is a small change, but might have a huge impact!
By default the ktp.next always outputs the same key. I saw multiple workarounds throughout this project like using the kpt.plaintext_next as a random number generator for both key and plaintext, but I'm not sure if there are parts that amuse default behavior....

We should discuss this change.

Reply:
This will be resolved in https://github.com/lowRISC/ot-sca/issues/116
For this PR we will use a workaround with backwards compatibility.